rubrik_cloudout_consolidation.template

```diff
@@ -0,0 +1,511 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Complete the AWS configuration process required for the Rubrik CloudOut with consolidation. CloudOut uses an S3 bucket for archiving to Amazon. When consolidation for Amazon S3 is enabled, the snapshot consolidation runs in the AWS cloud using the cloud compute resources.",
+  "Metadata": {
+    "AWS::CloudFormation::Interface": {
+      "ParameterGroups": [
+        {
+          "Label": { "default": "Storage Configuration" },
+          "Parameters": ["CreateNewS3Bucket", "S3BucketName"]
+        },
+        {
+          "Label": { "default": "Network Configuration" },
+          "Parameters": ["VPC",
+            "Subnet",
+            "OnPremRubrikCIDR",
+            "SecurityGroupName",
+            "SecurityGroupDescription",
+            "SecurityGroupRoleDescription"
+          ]
+        },
+        {
+          "Label": { "default": "IAM Users and Roles" },
+          "Parameters": ["CreateNewUser",
+            "IAMUserName"
+          ]
+        },
+        {
+          "Label": { "default": "KMS" },
+          "Parameters": [ "UseKMS" ]
+        }
+      ]
+    }
+  },
+
+  "Parameters": {
+    "CreateNewS3Bucket": {
+      "Description": "Create a new S3 Bucket to use as a Rubrik archival location.",
+      "Type": "String",
+      "Default": "no",
+      "AllowedValues": ["yes", "no"],
+      "ConstraintDescription": "Choose 'yes' to create a new S3 Bucket or 'no' to use an existing bucket."
+    },
+
+    "S3BucketName": {
+      "Description": "The name of the S3 Bucket to be used as a Rubrik archival location.",
+      "Type": "String",
+      "AllowedPattern": "^[a-z][a-z0-9_-]*[a-z0-9]${3,63}",
+      "ConstraintDescription": "Bucket name must be lowercase, contain only letters and numbers, not begin or end with special characters, and be 3-63 characters long"
+    },
+
+    "VPC": {
+      "Description": "Select the VPC for the archival location.",
+      "Type": "AWS::EC2::VPC::Id"
+    },
+
+    "Subnet": {
+      "Description": "Select the Subnet for the archival location.",
+      "Type": "AWS::EC2::Subnet::Id"
+    },
+
+    "OnPremRubrikCIDR": {
+      "Type": "String",
+      "Default": "10.79.0.0/24",
+      "Description": "The CIDR block for your on-premise Rubrik Cluster.",
```

Comment:
Change to:

"Description": "The CIDR block for your Rubrik Cluster that will use CloudOut."